docs/index.rst

```diff
@@ -73,38 +70,23 @@ Global Options
 --------------
 
 The ``rack`` CLI uses global options to alter output, authenticate, or
-pass in other **global** information into the tool. These options are
-called global because they are valid for any command. To see these, see :ref:`global_options`.
+pass in other global information into the tool. These options are
```

Comment:
"pass in other global information." or "pass other global information into the tool." (probably my mistake)